### DISCUSSION TITLE
Can't sync

### DISCUSSION THREAD

**Discussion by CaptainCrypt** [Chia-Network/chia-blockchain#5907] [Support]
On an iMac with port 8444 open and running version 1.1.6.  Haven't been able to sync in about 5 days.  Someone on keybase suggested I share these logs.  Had zero issues farming and plotting until about 5 days ago.  Stuck at 305,234.  Added main node as peer and done everything I know to do.  Have uninstalled/reinstalled multiple times.  Ideas? 

![Image 5-23-21 at 7 18 PM 2](https://user-images.githubusercontent.com/34175692/119281615-f82fce00-bbfb-11eb-8d64-5c863671cb07.jpg)
![Image 5-23-21 at 7 18 PM 3](https://user-images.githubusercontent.com/34175692/119281616-f8c86480-bbfb-11eb-8fce-b2943eb72483.jpg)
![Image 5-23-21 at 7 18 PM](https://user-images.githubusercontent.com/34175692/119281618-f960fb00-bbfb-11eb-99a8-66814bd18bd3.jpg)

**Comment by tiny0little** [comment]
inbound tcp/8444?

more ideas here - https://github.com/Chia-Network/chia-blockchain/wiki/Resolving-Sync-Issues---Port-8444

**Reply by CaptainCrypt** [reply]
8444 is open.  Did that when I first started farming and worked fine.  Ran port checker  and  it confirms 8444 is open.

did everything else in that link.  Logs make me feel like something maybe missing.

**Reply by tiny0little** [reply]
try adding full node and introducers manually

chia show -a node.chia.net:8444
chia show -a introducer-apne.chia.net:8444
chia show -a introducer-apse.chia.net:8444
chia show -a introducer-or.chia.net:8444
chia show -a introducer-va.chia.net:8444
chia show -a introducer-eu.chia.net:8444


make sure they are added

chia show -c| grep FULL_NODE

**Comment by CaptainCrypt** [comment]
Could something in these logs be the issue?  Is there a missing file or directory that is not installing on this version causing the issue?  Here's the logs I'm getting:

May 24 10:26:32 Marks-iMac Chia[8171]: getattrlist failed for /Library/GPUBundles/AMDRadeonX4000GLDriver.bundle/Contents/MacOS/ATIRadeonX4000SCLib.dylib: #2: No such file or directory
May 24 10:26:32 Marks-iMac Chia[8171]: getattrlist failed for /System/Library/Extensions/AMDRadeonX4000GLDriver.bundle/Contents/MacOS/ATIRadeonX4000SCLib.dylib: #2: No such file or directory
May 24 10:26:32 Marks-iMac Chia[8171]: getattrlist failed for /System/Library/Frameworks/OpenGL.framework/Resources//GLRendererFloat.bundle/GLRendererFloat: #2: No such file or directory
May 24 14:49:26 Marks-iMac Chia Helper (GPU)[31110]: getattrlist failed for /Library/GPUBundles/AMDRadeonX4000GLDriver.bundle/Contents/MacOS/ATIRadeonX4000SCLib.dylib: #2: No such file or directory
May 24 14:49:26 Marks-iMac Chia Helper (GPU)[31110]: getattrlist failed for /System/Library/Extensions/AMDRadeonX4000GLDriver.bundle/Contents/MacOS/ATIRadeonX4000SCLib.dylib: #2: No such file or directory
May 24 14:49:26 Marks-iMac Chia Helper (GPU)[31110]: getattrlist failed for /System/Library/Frameworks/OpenGL.framework/Resources//GLRendererFloat.bundle/GLRendererFloat: #2: No such file or directory

**Comment by Tofandel** [comment]
I have the same issue but only when the wallet is running, basically when the wallet is running the sync stops, if I stop the wallet the full node sync is ok.. When I start the wallet the full node sync stops being synced.. Can't farm that way.. It looks like there is some conflict between the full node of the wallet and the normal full node that wasn't happening pre 1.1.6

**Comment by CaptainCrypt** [comment]
My issue actually started before 1.1.6.  I was hoping updating to newest version would help, but it did not.  No idea what to do now. I spent money on a rig to farm and it is worthless until I fix this.

**Reply by Tofandel** [reply]
Well it seems you're syncing, but TBH it seems the network is at an uncontrollable growth right now and I think there is just more blocks being created than what the syncing process can handle, which is crazy (I think this project is going to be doomed).. I sync 50 blocks every minute but there is actually like 70 more being created in that time, which means for the time being it's impossible to catch up with the network

**Reply by Tofandel** [reply]
If you really have no movement, try to resync your system's clock

**Comment by CaptainCrypt** [comment]
Done everything that has been suggested and still no luck.  Stuck at 305,234.  Connecting with lots of peers at greater height, but can't sync.  

Is this just me or anyone else having issues?  Any other ideas?

**Comment by BrandtH22** [comment]
@CaptainCrypt , if you are still experiencing the issues it is recommended to upgrade to the latest version of chia. Please download it from here: https://www.chia.net/downloads/

Since this issue has been open for a number of years without additional comments, we will be closing this ticket but if we have closed this ticket in error do not hesitate to reach out to us again with any followup questions, comments, or if the issue persists after an update.

```
Please note that upgrading major versions of chia generally requires the config.yaml to be recreated using these steps:
1. Stop chia
2. Rename the config or move it to a different directory
3. In CLI run chia init
4. Copy any custom settings from the old config to the new (plot drive directories, pool info, trusted node info, etc) - be careful here as yaml is very picky about formatting including those leading spaces
5. Start Chia
```

The best place to reach our support team is on Discord (https://discord.gg/chia) or by reopening this ticket.
